### PR TITLE
make edit summaries more elaborate

### PR DESCRIPTION
**Pull request by rybak**
Add more context to the fixed edit summaries "Created by theusafBOT" and "Changed by theusafBOT".

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -220,20 +220,22 @@ async function createNewExplanation(info) {
 
     // If the comic title is a number underneath the current comic number, do not create this redirect
     if (!/^\d+$/.test(comicTitle) || +comicTitle > comicNum) {
+      const comicTitleRedirect = `#REDIRECT [[${comicNum}: ${comicTitle}]]\n`;
       await bot.edit(
         comicTitle,
-        `#REDIRECT [[${comicNum}: ${comicTitle}]]\n`,
-        EDIT_SUMMARY,
+        content,
```
Comment (theusaf) [inline index.js:226]: What does `content` refer to here?

File: index.js [hunk 2]
```diff
@@ -220,20 +220,22 @@ async function createNewExplanation(info) {
...
+        EDIT_SUMMARY + comicTitleRedirect,
```
Comment (theusaf) [inline index.js:227]: For these, could you use string templates rather than addition?
```js
`${EDIT_SUMMARY}${comicTitleRedirect}`
```

**Review by theusaf — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (rybak) [reply index.js]: This was a messed up variable renaming. Fixed in 740f5e9.

File: index.js (hunk 2 above)
Comment (rybak) [reply index.js]: Thanks. Applied the string template change in 740f5e9.

**Review by theusaf — CHANGES_REQUESTED** [review]
Could you apply the string templates to the other changes?

**Comment by theusaf** [comment]
Thanks!

**Comment by rybak** [comment]
Effects can be seen in [these five edits by the bot](https://www.explainxkcd.com/wiki/index.php?limit=50&title=Special%3AContributions&contribs=user&target=TheusafBOT&namespace=6&nsInvert=1&tagfilter=&start=2024-03-18&end=2024-03-18) on 2024-03-18.

demonstration of the effects of this PR

**Pull request merged**